.github/workflows/ubuntu_build_android.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - develop
+      - feature/tree_inventory
```

Comment:
Please remove this again before this branch gets merged.

Reply:
these has also been removed